src/InteractivePathAnalysis/NCriticalPathWidget.cpp

```diff
@@ -34,34 +34,30 @@
 #include <QPushButton>
 #include <QVBoxLayout>
 
+#ifndef TODO_IPA_MIGRATION_SETTINGS
+#include "../Compiler/QLSettingsManager.h"

```

Comment:
I don't see this file in the list. It looks like build will be broken with this flag.

Reply:
Closing this.